controllers/k8ssandra/k8ssandracluster_controller_test.go

```diff
@@ -105,8 +105,9 @@ func TestK8ssandraCluster(t *testing.T) {
 	t.Run("CreateMultiDcClusterWithMedusa", testEnv.ControllerTest(ctx, createMultiDcClusterWithMedusa))
 	t.Run("CreateSingleDcClusterNoAuth", testEnv.ControllerTest(ctx, createSingleDcClusterNoAuth))
 	t.Run("CreateSingleDcClusterAuth", testEnv.ControllerTest(ctx, createSingleDcClusterAuth))
-	// If webhooks are installed, this testcase is handled by the webhook test
+	// If webhooks are installed, these test cases are handled by the webhook test
 	// t.Run("ChangeNumTokensValue", testEnv.ControllerTest(ctx, changeNumTokensValue))
+	// t.Run("ChangeRackName", testEnv.ControllerTest(ctx, changeRackName))
```

Comment:
Note: I'm still having trouble running this locally (I commented out the webhook code to reach it, but it's giving me an error).

Reply:
It's fixed now.